### PR TITLE
feat: add hideKeyboard event

### PR DESCRIPTION
**Pull request by depapp**
hi @dmitry-zaitsev 

## description

I have an issue when try to `tapOn` a button that blocked by device keyboard.

- the UI before keyboard is showing
image

- the UI after keyboard is showing
image

## solution

I've tried to add `hideKeyboard()` event to avoid that failed case (before `tapOn()` desired button)

### CODE REVIEW

File: maestro-client/src/main/java/maestro/drivers/IOSDriver.kt [hunk 1]
```diff
@@ -220,6 +220,8 @@ class IOSDriver(
 
     override fun backPress() {}
 
+    override fun hideKeyboard() {}
```
Comment (dmitry-zaitsev) [inline maestro-client/src/main/java/maestro/drivers/IOSDriver.kt:223]: I just checked and it seems that pressing Escape key (key code `41`) on iOS is hiding the keyboard (at least in Safari). Let's use that

File: maestro-test/src/main/kotlin/maestro/test/drivers/FakeDriver.kt [hunk 2]
```diff
@@ -162,6 +162,12 @@ class FakeDriver : Driver {
         events += Event.BackPress
     }
 
+    override fun hideKeyboard() {
```
Comment (dmitry-zaitsev) [inline maestro-test/src/main/kotlin/maestro/test/drivers/FakeDriver.kt:165]: Could you please add a new test case in `IntegrationTest`?

Same pattern as with other tests there:
- Create a `yaml` file where you invoke `hide keyboard`
- Load this file in a test
- Run it on a `FakeDriver`
- Assert that `HideKeyboard` event was received

**Review by dmitry-zaitsev — CHANGES_REQUESTED** [review]
Looks good, but please also add:

- iOS support as mentioned in a comment
- Integration test

File: maestro-client/src/main/java/maestro/drivers/IOSDriver.kt (hunk 1 above)
Comment (depapp) [reply maestro-client/src/main/java/maestro/drivers/IOSDriver.kt]: updated on [27539f2](https://github.com/mobile-dev-inc/maestro/pull/165/commits/27539f2b48a65a80b12a6e3a2cff4b452ef92650)

File: maestro-test/src/main/kotlin/maestro/test/drivers/FakeDriver.kt (hunk 2 above)
Comment (depapp) [reply maestro-test/src/main/kotlin/maestro/test/drivers/FakeDriver.kt:165]: updated on this commit [fbc5007](https://github.com/mobile-dev-inc/maestro/pull/165/commits/fbc50075fbf271eb763cb3daf07cb44d47a574f0)

**Comment by depapp** [comment]
> Looks good, but please also add:
> 
> * iOS support as mentioned in a comment
> * Integration test

hi @dmitry-zaitsev 

I've updated the PR. could you please help to review it again? 🙇🏻

**Review by dmitry-zaitsev — APPROVED** [review]
Looks good! Thank you

**Pull request merged**